### PR TITLE
Coffee index page

### PR DESCRIPTION
**Pull request by timTonelli**
* created migration for coffee shop table
* created model for coffee shop
* created seeder for coffee shops table in database
* created index page on front end to display coffee shops
* created coffee shop tile component for index page to render for each coffee shop

### CODE REVIEW

**Review by ohad-porat — CHANGES_REQUESTED** [review]

**Comment by timTonelli** [comment]
Hi Ohad we made the suggested changes

File: client/src/components/CoffeeShopTile.js [hunk 1]
```diff
@@ -1,26 +1,18 @@
...
   return (
     <div>
       <h5>{name}</h5>
       <ul>
         <li>{city}</li>
         <li>{hours}</li>
-        {wifiItem}
-        {parkingItem}
+        {wifi ? <li>Wifi Available</li> : <li>No Wifi</li>}
+        {parking ? <li>Parking Available</li> : <li>No Parking</li>}
```
Comment (ohad-porat) [inline client/src/components/CoffeeShopTile.js:13]: I see you guys a getting nifty with these ternary operators, very cool!
I would suggest taking the logic out of the return statement though. We want to keep our JSX as clean and readable as possible.
Instead, you can set the ternary operator to a variable and then render it, which will also resolve the `<li>` repetition
```
const wifiAvailability = wifi ? "Wifi Available" : "No Wifi"
....
return
...<li>{wifiAvailability}</li>
```

**Review by ohad-porat — CHANGES_REQUESTED** [review]

**Comment by timTonelli** [comment]
Addressed this change, Ohad

File: server/src/models/CoffeeShop.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const Model = require("./Model");
+
+class CoffeeShop extends Model {
+  static get tableName() {
+    return "coffeeShops";
+  }
```
Comment (nwalberts) [inline server/src/models/CoffeeShop.js:6]: lets add an empty line after line 6 to space things out

File: server/src/routes/clientRouter.js [hunk 3]
```diff
@@ -3,9 +3,13 @@ import getClientIndexPath from "../config/getClientIndexPath.js";
...
+const clientRoutes = ["/","/coffee-shops", "/user-sessions/new", "/users/new"];
 router.get(clientRoutes, (req, res) => {
   res.sendFile(getClientIndexPath());
 });
 
 export default router;
+
+
+
+
```
Comment (nwalberts) [inline server/src/routes/clientRouter.js:15]: lets delete these extra lines

File: server/src/routes/rootRouter.js [hunk 4]
```diff
@@ -2,10 +2,18 @@ import express from "express";
 import userSessionsRouter from "./api/v1/userSessionsRouter.js";
 import usersRouter from "./api/v1/usersRouter.js";
 import clientRouter from "./clientRouter.js";
+import coffeeShopsRouter from "./api/v1/coffeeShopsRouter.js"
+
+
+
```
Comment (nwalberts) [inline server/src/routes/rootRouter.js:8]: delete extra lines

File: server/src/routes/rootRouter.js [hunk 5]
```diff
@@ -2,10 +2,18 @@ import express from "express";
...
 
 rootRouter.use("/api/v1/user-sessions", userSessionsRouter);
-rootRouter.use("/api/v1/users", usersRouter); //place your server-side routes here
+rootRouter.use("/api/v1/users", usersRouter); 
+rootRouter.use("/api/v1/coffee-shops", coffeeShopsRouter)
+//place your server-side routes here
 
 export default rootRouter;
+
+
```
Comment (nwalberts) [inline server/src/routes/rootRouter.js:19]: delete extra lines and code comment on line 15

**Review by nwalberts — APPROVED** [review]

**Pull request merged**